EcoSoapBank/Model/Payment.swift

```diff
@@ -12,8 +12,8 @@ struct Payment: Decodable {
     let id: String
     let invoiceCode: String?
     let invoice: String?
-    let amountPaid: String
-    let amountDue: String
+    let amountPaid: Int
+    let amountDue: Int
     let date: Date
```

Comment:
Yeah, I realized this when setting up my mock data. Hopefully both of us changing this independently won't cause a conflict.